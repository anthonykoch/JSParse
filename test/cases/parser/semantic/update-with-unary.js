+user--
-user--
+user++
-user++
+user.name--
-user.name--
+user.name++
-user.name++
