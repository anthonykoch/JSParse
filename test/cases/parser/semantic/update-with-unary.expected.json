---
name: semantic - update with unary
description: update with unary is allowed
skip: true
---
